src/Components/ContactUserCard.js

```diff
@@ -0,0 +1,3 @@
+let ContactUserCard = `ContactUserCard`;
```

Comment:
This is not a valid react component. A component should be a function that returns a JSX expression